src/base/static/client/mapseed-api-client/place.js

```diff
@@ -72,9 +72,35 @@ const createPlace = async ({
 
   const feature = await response.json();
 
-  return fromGeoJSONFeature({ feature, datasetSlug, clientSlug });
+  return {
+    place: fromGeoJSONFeature({ feature, datasetSlug, clientSlug }),
+    placeGeoJSON: {
+      ...feature,
+      properties: {
+        ...feature.properties,
+        _clientSlug: clientSlug,
+        _datasetSlug: datasetSlug,
+      },
+    },
+  };
 };
 
+const finalizePlaceData = ({ featureCollection, datasetSlug, clientSlug }) => ({
+  places: fromGeoJSONFeatureCollection({
+    featureCollection,
+    datasetSlug,
+    clientSlug,
+  }),
+  placesGeoJSONFeatures: featureCollection.features.map(feature => ({
```

Comment:
I think this is a code smell because now we are returning 2 representations of our places from the api client, and only sometimes we are doing it.

I would prefer returning the places as we did before, and if we need the geojson representation, we can use our serializer function to convert the place back to geojson. I understand it's not as performant, but I would favor that over the extra complexity.

What do you think?

Reply:
I was wondering about that too-- I'm happy to switch that around so we don't return two separate representations of the data from the API client.

Reply:
Yeah, I would prefer only returning a single representation of that data, and having the caller update the data format as needed.